### PR TITLE
[Tools] Delete Timepoint Script

### PR DESCRIPTION
**Pull request by stellarxo**
Creation of a new script that allows the deletion of a timepoint

Usage: php delete_timepoint.php delete_timepoint DCCID PSCID SessionID [confirm]
Example: php delete_timepoint.php delete_timepoint 965327 dcc0007 482
Example: php delete_timepoint.php delete_timepoint 965327 dcc0007 482 confirm

### CODE REVIEW

**Comment by stellarxo** [comment]
@ridz1208 please re-review :)

**Comment by ridz1208** [comment]
Should be runnable from any directory !!

**Comment by ridz1208** [comment]
Foreign key failure on media

```
Deleting from session.
PHP Warning:  PDO::exec(): SQLSTATE[23000]: Integrity constraint violation: 1451 Cannot delete or update a parent row: a foreign key constraint fails (`ccna_laptop`.`media`, CONSTRAINT `media_ibfk_1` FOREIGN KEY (`session_id`) REFERENCES `session` (`ID`)) in /var/www/loris/php/libraries/Database.class.inc on line 513
Could not execute DELETE FROM session WHERE `ID`='15'. Stack trace#0 /var/www/loris/tools/delete_timepoint.php(188): Database->delete('session', '`ID`='15'')
#1 /var/www/loris/tools/delete_timepoint.php(95): deleteTimepoint('15', true, Object(Database))
#2 {main}
PHP Fatal error:  Uncaught DatabaseException: Database DELETE did not execute successfully.Array
(
    [0] => 23000
    [1] => 1451
    [2] => Cannot delete or update a parent row: a foreign key constraint fails (`ccna_laptop`.`media`, CONSTRAINT `media_ibfk_1` FOREIGN KEY (`session_id`) REFERENCES `session` (`ID`))
)
 in /var/www/loris/php/libraries/Database.class.inc:515
Stack trace:
#0 /var/www/loris/tools/delete_timepoint.php(188): Database->delete('session', '`ID`='15'')
#1 /var/www/loris/tools/delete_timepoint.php(95): deleteTimepoint('15', true, Object(Database))
#2 {main}
  thrown in /var/www/loris/php/libraries/Database.class.inc on line 515
```

File: tools/delete_timepoint.php [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+            array('fid' => $id['FeedbackID'])
+        );
+        print_r($result);
+    }
+
+    // IF CONFIRMED, DELETE TIMEPOINT
+    if ($confirm) {
+        // Delete each instrument instance
+        foreach ($instruments as $instrument) {
+            $name = implode($instrument);
```
Comment (ridz1208) [inline tools/delete_timepoint.php]: implode needs "glue" the default is the empty string

**Comment by ridz1208** [comment]
@driusan to be closed

**Comment by alisterdev** [comment]
Rida made a new PR based on these changes which was tested and approved https://github.com/aces/Loris/pull/2548

**Pull request closed without merging**